common/transport/amqp/src/amqp.ts

```diff
@@ -14,7 +14,7 @@ import { AmqpLink } from './amqp_link_interface';
 
 import * as dbg from 'debug';
 import * as async from 'async';
-const debug = dbg('amqp-common:amqp');
+const debug = dbg('azure-iot-amqp-base:Amqp');
```

Comment:
>'azure-iot-amqp-base:Amqp [](start = 18, length = 25)

I'm glad you're doing this.  I've been using this facility and I appreciate the consistency.